src/test/java/com/microsoft/sqlserver/testframework/sqlType/SqlDateTime.java

```diff
@@ -41,4 +42,22 @@ public SqlDateTime() {
     public Object createdata() {
         return new Timestamp(ThreadLocalRandom.current().nextLong(((Timestamp) minvalue).getTime(), ((Timestamp) maxvalue).getTime()));
     }
+    
+    protected int generateRandomInt(int length) {
+        Random rnd = new Random();
+        int power = (int) Math.pow(10, length - 1);
+        
+        //Example of how this works:
+        // if length is 3, then we add 100 + random number between 0~899, so that we get a random number between 100~999.
+        int randomNumeric;
+        if (length <= 0) {
```

Comment:
nitpicking....
we normally do (0 >= blah) etc...

Reply:
i think that's only for comparing to null (for good reasons)